test-toolkit/resources/mnist/model_eia/mnist.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+# This file is intentionally left blank to utilize default_model_fn and default_predict_fn
```

Comment:
I'd put a newline before line 13 just so that it's easier to tell there's a real comment after the license header. Also, maybe rename this file so that it's obvious it's empty.

Reply:
bump